config/apinetlet/default/kustomization.yaml

```diff
@@ -15,6 +15,7 @@ namePrefix: apinetlet-
 resources:
 - ../rbac
 - ../manager
+- ../apinet-rbac
```

Comment:
@afritzler I am not sure if this should be employed here - i can remember this should only be done when all components run together in the same cluster.

If this is the case, the inclusion of `./apinet-rbac` should be done downstream as well depending of this, e.g. in the kind-specific kustomization configuration:

```
resources:
  - github.com/ironcore-dev/ironcore-net/config/apinetlet/apinet-rbac?ref=437081e6eea333713fa75ce6ccf3a46025a8dac0
  - github.com/ironcore-dev/ironcore-net/config/apinetlet/default?ref=437081e6eea333713fa75ce6ccf3a46025a8dac0
```

The same goes for `metalnetlet-rbac`

Reply:
Addressed